src/test/unit/test/utils/healthOptions.ts

```diff
@@ -1,7 +1,7 @@
 import { healthOptions } from '../../../../main/utils/healthOptions';
 import config from 'config';
 
-const mockResponse = () => {
+const mockResponseGood = () => {
```

Comment:
i think you can keep this as mockResponse and change the value for status in you method

Reply:
that way we dont have similar objects.

Reply:
also can leave status empty and change it good for the good response as well